.idea/libraries/Maven__org_mockito_mockito_all_1_9_5.xml

```diff
@@ -0,0 +1,13 @@
+<component name="libraryTable">
+  <library name="Maven: org.mockito:mockito-all:1.9.5">
+    <CLASSES>
+      <root url="jar://$MAVEN_REPOSITORY$/org/mockito/mockito-all/1.9.5/mockito-all-1.9.5.jar!/" />
+    </CLASSES>
+    <JAVADOC>
+      <root url="jar://$MAVEN_REPOSITORY$/org/mockito/mockito-all/1.9.5/mockito-all-1.9.5-javadoc.jar!/" />
+    </JAVADOC>
+    <SOURCES>
+      <root url="jar://$MAVEN_REPOSITORY$/org/mockito/mockito-all/1.9.5/mockito-all-1.9.5-sources.jar!/" />
+    </SOURCES>
+  </library>
+</component>
```

Comment:
I have this file since mockito has been added to the project.
There's an open issue as far as intellij files are concerned (#235).
Maybe could we keep this file and wait for the issue #235 to be managed ?
BTW, I added a unit test in my PR (to legitimate this file ? ;-) )